tests/ui/neg_multiply.rs

```diff
@@ -1,3 +1,4 @@
+// run-rustfix
 #![warn(clippy::neg_multiply)]
 #![allow(clippy::no_effect, clippy::unnecessary_operation)]
```

Comment:
```suggestion
#![allow(clippy::no_effect, clippy::unnecessary_operation, clippy::precedence)]
```
The `clippy::precedence` lint triggers for the `0xcafe | 0xff00 * -1;` expression. This causes the pipeline to fail. If you allow that lint and rerun the tests and update the output, everything should be fine :upside_down_face:

Reply:
Done 😃

Reply:
The pipeline fails because it can't compile the fixed code. I tested it locally, it compiles fine 😕

Reply:
Some tests can't be executed on Windows, if you use that it might explain this behavior. I'll check it out and try to find the problem :upside_down_face:

Reply:
The tests fail due to the configuration of `compiletest-rs`. In the normal UI tests all lints inside the `unused` group are allowed, this is sadly not true for `.fixed` files. The problem can be solved by adding `#![allow(unused)]` to the file, as that will allow these lints and prevent the warnings that make the test fail.

(This is something I want to fix at some point, but for now, this is the best and easiest solution. Sorry that you ran into this :sweat_smile:)

Reply:
> Some tests can't be executed on Windows, if you use that it might explain this behavior. I'll check it out and try to find the problem 🙃

Yes, I am on Windows, but actually running the test locally was failing too. I thought the problem was about compiling the .fixed file.





> The tests fail due to the configuration of `compiletest-rs`. In the normal UI tests all lints inside the `unused` group are allowed, this is sadly not true for `.fixed` files. The problem can be solved by adding `#![allow(unused)]` to the file, as that will allow these lints and prevent the warnings that make the test fail.

I was not sure if the `#![allow(unused)]` should be added in the .fixed only even though it's a generated file ?
I added it in test file for now.

Reply:
> Yes, I am on Windows, but actually running the test locally was failing too. I thought the problem was about compiling the .fixed file.

Alright :+1:, your assumption was correct, it was with the compilation of the `.fixed` files. 

> I was not sure if the `#![allow(unused)]` should be added in the .fixed only even though it's a generated file?
I added it in test file for now.

You did it correctly, that's what I meant :+1: